dlrover/python/elastic_agent/torch/training.py

```diff
@@ -59,20 +58,16 @@
     TrainingMsgLevel,
 )
 from dlrover.python.common.log import default_logger as logger
+from dlrover.python.elastic_agent.config.paral_config_tuner import (
+    ParalConfigTuner,
+)
 from dlrover.python.elastic_agent.master_client import GlobalMasterClient
 from dlrover.python.elastic_agent.monitor.training import TorchTrainingMonitor
 from dlrover.python.elastic_agent.torch.master_kv_store import MasterKVStore
 
 __all__ = ["launch_agent"]
 
 
-def _set_paral_config():
-    config_dir = os.path.dirname(ConfigPath.PARAL_CONFIG)
-    os.makedirs(config_dir, exist_ok=True)
-    os.environ[ConfigPath.ENV_PARAL_CONFIG] = ConfigPath.PARAL_CONFIG
-    os.environ[ConfigPath.ENV_RUNTIME_METRICS] = ConfigPath.RUNTIME_METRICS
```

Comment:
Keep the line because the training process needs to save metrics into a JSON file and the main process reads the file to report the metrics to the master.

Reply:
Updated in latest commit.